backend/main.py

```
@@ -175,15 +175,19 @@
 # Import database initialization
 from init_database import initialize_jyotiflow_database
 
-# Enhanced startup integration and fixes are now consolidated in unified_startup_system.py
+# Enhanced startup integration and fixes are now consolidated in simple_unified_startup.py
 
 # Import database schema fix
 from db_schema_fix import fix_database_schema
```

Ruff (0.12.2)

`db_schema_fix.fix_database_schema` imported but unused

Remove unused import: `db_schema_fix.fix_database_schema`

(F401)